Controller/Annotations/AbstractScalarParam.php

```diff
@@ -63,6 +64,12 @@ public function getConstraints()
             $constraints = array(
                 new All(array('constraints' => $constraints)),
             );
+            if (false === $this->nullable) {
+                $constraints[] = new NotNull();
+            }
+            if (false === $this->allowBlank) {
```

Comment:
I think we should move the `if` from line 57 after the `if ($this->map)` block instead.

Reply:
see https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1970